src/app/styles/Home.module.scss

```diff
@@ -0,0 +1,282 @@
+
+.productOfferingsLink {
```

Comment:
I know you called the CSS out specifically but I'm good with what's here for now 👍 We can likely do some clean up as we keep making progress but I don't know that we necessarily need to tackle that in this PR unless we'd like to.